client/status/common.go

```diff
@@ -2,12 +2,39 @@ package apistatus
 
 import (
 	"encoding/binary"
+	"errors"
 
 	"github.com/nspcc-dev/neofs-api-go/v2/status"
 )
 
+// ErrAPI describes common error which is a grouping type for any [apistatus] errors. Any [apistatus] error may be checked
+// explicitly via it's type of just check the group via errors.Is(err, [apistatus.ErrAPI]).
+var ErrAPI = errors.New("api error")
+
+var (
+	// ErrServerInternal is an instance of ServerInternal error status. It's expected to be used for [errors.Is]
+	// and MUST NOT be changed.
+	ErrServerInternal ServerInternal
+	// ErrWrongMagicNumber is an instance of WrongMagicNumber error status. It's expected to be used for [errors.Is]
+	// and MUST NOT be changed.
+	ErrWrongMagicNumber WrongMagicNumber
+	// ErrSignatureVerification is an instance of SignatureVerification error status. It's expected to be used for [errors.Is]
+	// and MUST NOT be changed.
+	ErrSignatureVerification SignatureVerification
+	// ErrNodeUnderMaintenance is an instance of NodeUnderMaintenance error status. It's expected to be used for [errors.Is]
+	// and MUST NOT be changed.
+	ErrNodeUnderMaintenance NodeUnderMaintenance
+)
+
+// APIErrorDetails gives unified method to get details and determine what happened from certain error.
+type APIErrorDetails interface {
```

Comment:
`StatusV2` isn't enough for this? This interface describes version-dependent elements, no wonder you have to import `neofs-api-go/v2`

Reply:
@roman-khimov What do you think about @cthulhu-rider suggestion? I mean remove APIErrorDetails and use `target.ToStatusV2()` instead of it, if required?

Reply:
OK, let's drop `APIErrorDetails` and rely on `ToStatusV2`.

Reply:
removed